packages/web3/src/entitlement.ts

```diff
@@ -65,7 +67,8 @@ export type CheckOperation = {
     checkType: CheckOperationType
     chainId: bigint
     contractAddress: Address
-    threshold: bigint
+    threshold?: bigint
```

Comment:
This should be params bytes, not specific data, so we never have to go through this again for new operation types.